pallets/elections/src/lib.rs

```diff
@@ -1,6 +1,22 @@
 //! This pallet manages changes in the committee responsible for producing blocks and establishing consensus.
 //! Currently, it's PoA where the validators are set by the root account. In the future, a new
 //! version for DPoS elections will replace the current one.
+//!
+//! ### Terminology
+//! For definition of session, era, staking see pallet_session and pallet_staking.
+//! - Committee: Set of nodes that produce and finalize blocks in the era.
+//! - Validator: Node that can become a member of committee (or already is) via rotation.
+//! - (TODO: remove this to remove confusion) Member: Usually same as validator, sometimes means member of the committee
+//! - ReservedMembers: Validators that are chosen to be in committee every single session.
+//!
+//! ### Storage
+//! - `Members` - List of possible validators.
+//! - `MembersPerSession` - Number of committee size.
+//! - `ReservedMembers` - List of reserved nodes.
+//! - `ErasReserved` - List of reserved nodes for the current era.
```

Comment:
does it mean that reserved members are (potentially) changed each era?

Reply:
yep

Reply:
I would add a word about this then

Reply:
the line 17 is not enough?